doc-content/jbpm-docs/src/main/asciidoc/BPMN2/create-service-task-proc.adoc

```diff
@@ -46,18 +61,114 @@ A service task is a task that is part of the process that is executed outside of
 | *Is Multiple Instance*
 | Select if this task has multiple instances.
 
+| *MI Execution mode*
+| Select if the multiple instances execute in parallel or sequentially.
+
+| *MI Collection input*
+| Specify a variable that represents a collection of elements for which new instances are created, such as `inputCountryNames`.
+
+| *MI Data Input*
+| Specify the input data assignment that is transferred to a web service, such as `Parameter`.
+
+| *MI Collection output*
+| The array list in which values returned from the web service task is stored, such as `outputCountries`.
+
+| *MI Data Output*
+| Specify the output data assignment for the web service task, which stores the result of class execution on the server, such as `Result`.
+
+| *MI Completion Condition (mvel)*
+| Specify the MVEL expression that is evaluated on each completed instance to check if the specified multiple instance node can complete.
+
 | *On Entry Action*
 | A Java, JavaScript, or MVEL script that specifies an action at the start of the task.
 
 | *On Exit Action*
 | A Java, JavaScript, or MVEL script that specifies an action at the end of the task.
 
-
 | *SLA Due Date*
 | The date that the service level agreement (SLA) expires.
 
 
 |===
 
+. Click *Save*.
+
+When you create a service task using Java method, then the method can only contain one parameter and returns a single value. To create a service task using a Java method, you must add the Java class to the dependencies of the project:
+
+. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Artifacts*.
```

Comment:
Maybe how to add **Artifacts** already exists and we can just link it her to prevent duplications?

Reply:
@hasys, correct uploading Artifact is already mentioned in a user story but consists of a general approach. Here, we are following the procedure of adding .jar files and then adding dependencies in the project, so I believe user has a particular process at one place. 
wdyt?

Reply:
TBH I don't know style guide here, so just sharing my thoughts. If duplications is OK according to style guides, current form looks good to me, thanks :+1: